Makefile

```diff
@@ -1,8 +1,52 @@
-vendor: vendor/modules.txt
 
-vendor/modules.txt: go.mod
-	go mod vendor
+.PHONY: build help images push
 
-.PHONY: build
-build: vendor
-	go build
\ No newline at end of file
+
+ARCH ?= amd64
+BIN_NAME = clair-load-test
+BIN_DIR = bin
+BIN_PATH = $(BIN_DIR)/$(ARCH)/$(BIN_NAME)
+CGO = 0
+
+SOURCES := $(shell find . -type f -name "*.go")
+
+# Containers
+ENGINE ?= podman
+REGISTRY = quay.io
+ORG ?= vchalla
+CONTAINER_NAME_ARCH = $(REGISTRY)/$(ORG)/clair-load-test:$(ARCH)
+
+all: lint build images push
+
+help:
+	@echo "Commands for $(BIN_PATH):"
+	@echo
+	@echo 'Usage:'
+	@echo '    make clean                    Clean the compiled binaries'
+	@echo '    [ARCH=arch] make build        Compile the project for arch, default amd64'
+	@echo '    [ARCH=arch] make install      Installs clair-load-test binary in the system, default amd64'
+	@echo '    [ARCH=arch] make images       Build images for arch, default amd64'
+	@echo '    [ARCH=arch] make push         Push images for arch, default amd64'
+	@echo '    make help                     Show this message'
+
+build: $(BIN_PATH)
+
+$(BIN_PATH): $(SOURCES)
+	@echo -e "\033[2mBuilding $(BIN_PATH)\033[0m"
+	@echo "GOPATH=$(GOPATH)"
+	GOARCH=$(ARCH) CGO_ENABLED=$(CGO) go build -v -mod vendor -o $(BIN_PATH) ./cmd/clair-load-test
+
+lint:
+	find . -name '*.go' -type f -exec go fmt {} \;
+	golangci-lint run
+
+install:
+	cp $(BIN_PATH) /usr/bin/$(BIN_NAME)
```

Comment:
`install` also does this (and I believe has some permissions niceties)

Reply:
Yes. I have mentioned about this in `README.md`, simply adding a `sudo` should do it.